src/frequenz/channels/util/_periodic_timer.py

```diff
@@ -0,0 +1,201 @@
+# License: MIT
+# Copyright © 2022 Frequenz Energy-as-a-Service GmbH
+
+"""A timer receiver that returns the timestamp every `interval`."""
+
+from __future__ import annotations
+
+import asyncio
+from datetime import timedelta
+
+from .._base_classes import Receiver
+from .._exceptions import ReceiverStoppedError
+
+
+class PeriodicTimer(Receiver[timedelta]):
+    """A periodic timer receiver that fires every `interval` time.
+
+    The message it produces is a `timedelta` containing the drift of the timer,
+    i.e. the difference between when the timer should have fired and the time
+    when it actually fired. Note that the drift will not be accumulated by the
+    timer, which will be automatically adjusted so it only fires (or tries to)
+    in multiples of the `interval`.
+
+    This will not always be `0`, as if there is a task that is running when it
```

Comment:
```suggestion
    This will not always be `0`, because if there is a task that is running when it
```
because "as if" has another comparitive meaning, it is a bit confusing.